fjord/base/middleware.py

```diff
@@ -163,4 +163,5 @@ def process_request(self, request):
 
         request.path_info = '/' + prefixer.shortened_path
         request.locale = prefixer.locale
-        tower.activate(prefixer.locale)
+        translation.activate(prefixer.locale)
+        install_gettext()
```

Comment:
That sounds like it'd work.

I'm ok with it here, too. It's pretty menial. And if it goes away with django-jinja, then maybe it doesn't make sense to fiddle with things.